docs/mast/mast_mastquery.rst

```diff
@@ -3,26 +3,118 @@
 MAST Queries
 ************
 
-Direct Mast Queries
-===================
-
 The Mast class provides more direct access to the MAST interface.  It requires
 more knowledge of the inner workings of the MAST API, and should be rarely
 needed.  However in the case of new functionality not yet implemented in
 astroquery, this class does allow access.  See the `MAST api documentation
-<https://mast.stsci.edu/api>`_ for more information.
+<https://mast.stsci.edu/api>`__ for more information.
+
+The basic MAST query function accepts
+`MAST Service <https://mast.stsci.edu/api/v0/_services.html>`__-specific
```

Comment:
I would reword this slightly to: _The basic MAST query function allows users to query through the following `MAST Services <https://mast.stsci.edu/api/v0/_services.html>`__ using their corresponding parameters_ (with the exception of the service that you mentioned `mast_query` doesnt account for).